src/FSharpPlus/Extensions/List.fs

```diff
@@ -177,18 +173,12 @@ module List =
     /// <param name="lst">The input list</param>
     /// 
     /// <returns>For invalid indexes, the input list.  Otherwise, a new list with the item removed.</returns>
-    let removeAt i lst =
-        if List.length lst > i then
-            lst.[0..i-1] @ lst.[i+1..]
-        else lst
+    let removeAt i lst = FSharp.Collections.List.removeAt i lst
```

Comment:
> For invalid indexes, the input list.

No, the FSharp.core version throws.

Reply:
So should we keep this implementation and use an obsolete attribute

Reply:
IMHO: No, yes.

No, we need to rename it to something else (first think that comes to my mind is `deleteAt`) and 
yes, issue a deprecate warning, saying something like "This function was included in FSharp.Core but throwing,  use `deletaAt` (or whichever name we choose) instead ot if you want to throw exceptions use the full path to `removeAt` in Core ".